impress-org/give-email-reports
# fix(uninstall): get fatal error when this addon is active and deativating Give core #52

Closes #52 

## Description
This PR fixes the fatal error caused by a function that was inaccessible in Give Email Reports after deactivating Give Core.  

## How Has This Been Tested?
1. Make sure Give Core and Give Email Reports are both active.
2. Deactivate Give Email Reports.
3. Check debug log for errors.

## Checklist:
